bc_obps/service/tests/test_facility_designated_operation_timeline_service.py

```diff
@@ -47,19 +45,14 @@ def test_get_timeline_by_operation_id_industry_user():
         users_operation = baker.make_recipe(
             'registration.tests.utils.operation', operator=approved_user_operator.operator
         )
-        # user's timeline - 5 transferred, 10 active to make sure the filter is working
-        baker.make_recipe(
-            'registration.tests.utils.facility_designated_operation_timeline',
-            operation=users_operation,
-            _quantity=5,
-            status=FacilityDesignatedOperationTimeline.Statuses.TRANSFERRED,
-        )
+        # user's timeline
         baker.make_recipe(
             'registration.tests.utils.facility_designated_operation_timeline',
             operation=users_operation,
             _quantity=10,
-            status=FacilityDesignatedOperationTimeline.Statuses.ACTIVE,
         )
+        # mimic transferred facilities
```

Comment:
Missing code here? It looks like we might be missing the code to handle transferred facilities in the service too